SimObjects/Airplanes/Asobo_B747_8i/flight_model.cfg

```diff
@@ -280,11 +280,12 @@ Pump.24 = Name:ReservePump4Fwd#Pressure:10#DestinationLine:ReservePump4FwdToRese
-Trigger.3 = Target:CenterWingTank#Threshold:11940#Condition:TankQuantityBelow#EffectTrue:OpenValve.StabTankValve1,OpenValve.StabTankValve2,OpenValve.CenterWingStabJuncValve1,OpenValve.CenterWingStabJuncValve2,OpenValve.StabTransferValve1,OpenValve.StabTransferValve2#EffectFalse:CloseValve.StabTankValve1,CloseValve.StabTankValve2,CloseValve.CenterWingStabJuncValve1,CloseValve.CenterWingStabJuncValve2,CloseValve.StabTransferValve1,CloseValve.StabTransferValve2
-Trigger.4 = Target:MainTank1#Threshold:2015#Condition:TankQuantityBelow#EffectTrue:OpenValve.ReserveTank2ValveFwd,OpenValve.ReserveTank2ValveAft,StartPump.ReservePump1Fwd,StartPump.ReservePump1Aft
-Trigger.5 = Target:MainTank4#Threshold:2015#Condition:TankQuantityBelow#EffectTrue:OpenValve.ReserveTank3ValveFwd,OpenValve.ReserveTank3ValveAft,StartPump.ReservePump4Fwd,StartPump.ReservePump4Aft
-Trigger.6 = Target:ReserveTank2#Threshold:1#Condition:TankQuantityBelow#EffectTrue:StopPump.ReservePump1Fwd,StopPump.ReservePump1Aft,CloseValve.ReserveTank2ValveFwd,CloseValve.ReserveTank2ValveAft
-Trigger.7 = Target:ReserveTank3#Threshold:1#Condition:TankQuantityBelow#EffectTrue:StopPump.ReservePump4Fwd,StopPump.ReservePump4Aft,CloseValve.ReserveTank3ValveFwd,CloseValve.ReserveTank3ValveAft
+Trigger.3 = Target:CenterWingTank#Threshold:11940#Condition:TankQuantityBelow#EffectTrue:OpenValve.StabTankValve1,OpenValve.StabTankValve2,OpenValve.CenterWingStabJuncValve1,OpenValve.CenterWingStabJuncValve2,OpenValve.StabTransferValve1,OpenValve.StabTransferValve2
+Trigger.4 = Target:StabTank#Threshold:1#Condition:TankQuantityBelow#EffectTrue:CloseValve.StabTankValve1,CloseValve.StabTankValve2,CloseValve.CenterWingStabJuncValve1,CloseValve.CenterWingStabJuncValve2,CloseValve.StabTransferValve1,CloseValve.StabTransferValve2
+Trigger.5 = Target:MainTank1#Threshold:2015#Condition:TankQuantityBelow#EffectTrue:OpenValve.ReserveTank2ValveFwd,OpenValve.ReserveTank2ValveAft,StartPump.ReservePump1Fwd,StartPump.ReservePump1Aft
+Trigger.6 = Target:MainTank4#Threshold:2015#Condition:TankQuantityBelow#EffectTrue:OpenValve.ReserveTank3ValveFwd,OpenValve.ReserveTank3ValveAft,StartPump.ReservePump4Fwd,StartPump.ReservePump4Aft
+Trigger.7 = Target:ReserveTank2#Threshold:1#Condition:TankQuantityBelow#EffectTrue:StopPump.ReservePump1Fwd,StopPump.ReservePump1Aft,CloseValve.ReserveTank2ValveFwd,CloseValve.ReserveTank2ValveAft
+Trigger.8 = Target:ReserveTank3#Threshold:1#Condition:TankQuantityBelow#EffectTrue:StopPump.ReservePump4Fwd,StopPump.ReservePump4Aft,CloseValve.ReserveTank3ValveFwd,CloseValve.ReserveTank3ValveAft

```

Comment:
what exact changes does this introduce?

Reply:
Because the arrows show when the respective valves are opened, I had to add an extra trigger when the xfer is complete that was missing to close them, so that the arrows would be hidden again.